mautic/mautic
# Send email to DNC contacts in campaigns actions

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
The functionality allows you to send email messages in the action of the campaign "Send email to contacts" to people with the DNC flag. By default, this option is disabled

![image](https://user-images.githubusercontent.com/39382654/114534883-c23c1900-9c4f-11eb-8c93-70be0ba12840.png)




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Check the "Send to SYNC contacts" option to yes
3. Run the campaign action on the user who has the DNC flag, an email should be sent
